lib/puppet/type/bootstrap_rpm.rb

```diff
@@ -0,0 +1,19 @@
+Puppet::Type.newtype(:bootstrap_rpm) do
```

Comment:
This should have autorequires (https://puppet.com/docs/puppet/latest/custom_types.html#tandp_automatic_relationships):
```ruby
autorequire(:file) do
  [self[:dest]]
end

autorequire(:concat) do
  [self[:script]]
end
```